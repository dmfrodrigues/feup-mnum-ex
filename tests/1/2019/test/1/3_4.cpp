#include <bits/stdc++.h>
using namespace std;
double g3(double x){
  return log(x+11.0L);
}
int main(){
  double x = 2.0L, x_;
  cout << setprecision(20) << fixed;
  for(int i = 0; i < 1000; ++i){
    x_ = x;
    x = g3(x);
    cout << x << endl;
    if(x == x_) break;
  }
  return 0;
}
