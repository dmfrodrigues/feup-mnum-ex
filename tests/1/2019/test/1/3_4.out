2.56494935746153673861
2.60748921278004397450
2.61062031858513909910
2.61085039377205108124
2.61086729772183012699
2.61086853966768872581
2.61086863091432119788
2.61086863761827459030
2.61086863811081881792
2.61086863814700631536
2.61086863814966507746
2.61086863814986047672
2.61086863814987468757
2.61086863814987601984
2.61086863814987601984
